<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>account</class>
 <widget class="QDialog" name="account">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>90</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="middleName">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lastName">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="position">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>70</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="email">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>110</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="type_user">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>140</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>30</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
